### PR TITLE
translate 8 EBNF related PRs

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
   - https://github.com/pingcap/docs/pull/17070
   - https://github.com/pingcap/docs/pull/17072
   - https://github.com/pingcap/docs/pull/17074
   - https://github.com/pingcap/docs/pull/17081
   - https://github.com/pingcap/docs/pull/17082
   - https://github.com/pingcap/docs/pull/17083
   - https://github.com/pingcap/docs/pull/17087
   - https://github.com/pingcap/docs/pull/17091
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-show-create-user.md [hunk 1]
```diff
@@ -45,7 +42,7 @@ mysql> SHOW GRANTS FOR 'root';
 
 ## MySQL 兼容性
 
-* `SHOW CREATE USER` 的输出结果旨在匹配 MySQL，但 TiDB 尚不支持若干 `CREATE` 选项。尚未支持的选项在语句执行过程中会被解析但会被跳过执行。详情可参阅 [security compatibility]。
+* `SHOW CREATE USER` 的输出结果旨在匹配 MySQL，但 TiDB 尚不支持若干 `CREATE` 选项。尚未支持的选项在语句执行过程中会被解析但会被跳过执行。详情可参阅 [与 MySQL 安全特性差异](/security-compatibility-with-mysql.md)。
```
Comment (qiancai) [inline sql-statements/sql-statement-show-create-user.md:45]:
```suggestion
* `SHOW CREATE USER` 的输出结果旨在匹配 MySQL，但 TiDB 尚不支持若干 `CREATE` 选项。尚未支持的选项在语句执行过程中会被解析但会被跳过执行。详情可参阅[与 MySQL 安全特性差异](/security-compatibility-with-mysql.md)。
```

File: sql-statements/sql-statement-show-profiles.md [hunk 2]
```diff
@@ -10,14 +10,16 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-show-profiles/']
 
 ## 语法图
 
-**ShowStmt:**
-
-![ShowStmt](/media/sqlgram/ShowStmt.png)
+```ebnf+diagram
+ShowProfilesStmt ::=
+    "SHOW" "PROFILES" ShowLikeOrWhere?
+ShowLikeOrWhere ::=
```
Comment (hfxsd) [inline sql-statements/sql-statement-show-profiles.md:16]:
```suggestion

ShowLikeOrWhere ::=
```

File: sql-statements/sql-statement-set-names.md [hunk 3]
```diff
@@ -10,33 +10,10 @@ aliases: ['/docs-cn/dev/sql-statements/sql-statement-set-names/','/docs-cn/dev/r
...
-
-**CharsetNameOrDefault:**
-
-![CharsetNameOrDefault](/media/sqlgram/CharsetNameOrDefault.png)
+```ebnf+diagram
+SetNamesStmt ::=
+    "SET" ("NAMES" ("DEFAULT" | CharsetName ("COLLATE" ("DEFAULT" | CollationName))?) | ("CHARSET" | ("CHAR" | "CHARACTER") "SET") ("DEFAULT" | CharsetName))
+```
 
 ## 示例
```
Comment (hfxsd) [inline sql-statements/sql-statement-set-names.md:19]: 把 L20 的 {{< copyable "sql" >}} 也删掉

File: sql-statements/sql-statement-show-stats-healthy.md [hunk 4]
```diff
@@ -7,28 +7,23 @@ summary: TiDB 数据库中 SHOW STATS_HEALTHY 的使用概况。
 
 `SHOW STATS_HEALTHY` 语句可以预估统计信息的准确度，也就是健康度。健康度低的表可能会生成次优查询执行计划。
 
-可以通过执行 `ANALYZE` 表命令来改善表的健康度。当表的健康度下降到低于 [`tidb_auto_analyze_ratio`](/system-variables.md#tidb_auto_analyze_ratio) 时，则会自动执行 `ANALYZE` 命令。
+可以通过执行 [`ANALYZE`](/sql-statements/sql-statement-analyze-table.md) 语句来改善表的健康度。当表的健康度下降到低于 [`tidb_auto_analyze_ratio`](/system-variables.md#tidb_auto_analyze_ratio) 时，则会自动执行 `ANALYZE` 命令。
```
Comment (hfxsd) [inline sql-statements/sql-statement-show-stats-healthy.md:10]:
```suggestion
可以通过执行 [`ANALYZE`](/sql-statements/sql-statement-analyze-table.md) 语句来改善表的健康度。当表的健康度下降到低于 [`tidb_auto_analyze_ratio`](/system-variables.md#tidb_auto_analyze_ratio) 时，则会自动执行 `ANALYZE` 语句。
```

File: sql-statements/sql-statement-show-stats-healthy.md [hunk 5]
```diff
@@ -7,28 +7,23 @@ summary: TiDB 数据库中 SHOW STATS_HEALTHY 的使用概况。
...
-
-![ShowLikeOrWhereOpt](/media/sqlgram/ShowLikeOrWhereOpt.png)
+ShowLikeOrWhere ::=
+    "LIKE" SimpleExpr
+|   "WHERE" Expression
+```
 
 ## 示例
 
 加载示例数据并运行 `ANALYZE` 命令：
```
Comment (hfxsd) [inline sql-statements/sql-statement-show-stats-healthy.md:25]:
```suggestion
加载示例数据并运行 `ANALYZE` 语句：
```

**Review by hfxsd — APPROVED** [review]

File: sql-statements/sql-statement-set-names.md (hunk 3 above)
Comment (qiancai) [reply sql-statements/sql-statement-set-names.md:19]: Done: https://github.com/pingcap/docs-cn/pull/17216/commits/f9fe52c3d44d265dcc434340fdcfb68bee20b8bb

**Pull request merged**